src/v2/controllers/FormController.js

```diff
@@ -216,12 +218,14 @@ export default Controller.extend({
           onSuccess();
         }
       }).catch(error => {
-        if (error.stepUp) {
+        if (error.stepUp && !this.isBiometricsErrorFlow(error)) {
```

Comment:
we should not do this `!this.isBiometricsErrorFlow(error)` ... View specific code should remain in Views.